src/Application.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of DOCtor-RST.
+ *
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App;
+
+use App\Command\AnalyseCommand;
+use App\Command\RulesCommand;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Console\Application as BaseApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+class Application extends BaseApplication
+{
+    public const VERSION = '@git-version@';
```

Comment:
This is a placeholder for humbug/box https://github.com/humbug/box/blob/master/doc/configuration.md#git-version-placeholder-git-version and box is not a part of this PR